bans.txt

```diff
@@ -7,5 +7,4 @@ NzY1NjExOTkyNDEzNjM3ODA=
 NzY1NjExOTk2MDkzODU5OTk=
 NzY1NjExOTgxODI1NTU2NzU=
 NzY1NjExOTkxMjg5NjExNTE=
-NzY1NjExOTgwNTUzODIxMDI=
-NzY1NjExOTk2ODI1NDg1NTY=
```

Comment:
I think this fork was out of date since this was added. Can you pull in the latest bans.txt so this isn't removed?